src/lib.rs

```diff
@@ -771,6 +802,46 @@ impl UnixDatagram {
         }
     }
 
+    /// Receives data on the socket.
+    ///
+    /// If path is None, the peer address set by the `connect` method will be used.  If it has not
+    /// been set, then this method will return an error.
+    ///
+    /// This interface allows receiving data into multiple buffers.  This acts as if the buffers had been
+    /// concatenated in the order they were given.
+    ///
+    /// cmsg_buffer is space to use for storing control messages.
```

Comment:
This field seems a bit weird - it's basically scratch space for the internal logic, right?